src/Entity/DeliveryConfirmation.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use Ups\NodeInterface;
+
+/**
+ * Class DeliveryConfirmation
+ * @package Ups\Entity
+ */
+class DeliveryConfirmation implements NodeInterface
+{
+    /**
```

Comment:
Could you add some constants for 1 and 2? 

E.g.:

```
const DELIVERY_CONFIRMATION_SIGNATURE_REQUIRED = 1;
```

Allows for easier / readable usage inside code :)